### PR TITLE
Add checksum for py-terminado 0.15.0

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
LGTM.  Confirmed sha256.

File: var/spack/repos/builtin/packages/py-terminado/package.py [hunk 1]
```diff
@@ -12,16 +12,20 @@ class PyTerminado(PythonPackage):
...
     version("0.8.1", sha256="55abf9ade563b8f9be1f34e4233c7b7bde726059947a593322e8a553cc4c067a")
     version("0.6", sha256="2c0ba1f624067dccaaead7d2247cfe029806355cef124dc2ccb53c83229f0126")
 
+    depends_on("python@3.7:", when="@0.15.0:", type=("build", "run"))
     depends_on("python@3.6:", when="@0.12.1:", type=("build", "run"))
     depends_on("python@2.7:2.8,3.4:", when="@0.8.2:", type=("build", "run"))
-    depends_on("py-setuptools@40.8.0:", when="@0.12.1:", type="build")
-    depends_on("py-setuptools", type="build")
+    depends_on("py-setuptools@40.8.0:", when="@0.12.1", type="build")
+    depends_on("py-setuptools", when="@:0.12.1", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-terminado/package.py]:
```suggestion
    depends_on("py-setuptools", when="@:0.6", type="build")
```
0.8 doesn't actually depend on setuptools

**Review by adamjstewart — CHANGES_REQUESTED** [review]
This is the first package I've seen that's changed their build backend at least 4 times! Excited to see who the 5th will be.

* 0.6: distutils
* 0.8: flit
* 0.12: setuptools
* 0.15: hatchling

**Comment by iarspider** [comment]
I wonder why they went flit -> setuptools, not setuptools -> flit 🤔

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
> I wonder why they went flit -> setuptools, not setuptools -> flit 🤔

I'm guessing because the repo went from https://github.com/takluyver/terminado to https://github.com/jupyter/terminado (takluyver is the developer of flit)

**Pull request merged**